orchestration/templates/extract-from-json.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: extract-from-json
+spec:
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: pvc-name
+          - name: file-path
+          - name: json-key
+      volumes:
+        - name: state
+          persistentVolumeClaim:
+            claimName: '{{ "{{inputs.parameters.pvc-name}}" }}'
+            readOnly: true
+      {{- include "argo.retry" . | indent 6 }}
+      script:
+        image: python:3-slim
+        command: [python]
+        source: |
+          import json
+          with open("{{inputs.parameters.file-path}}") as f:
```

Needs to be rooted within the state volume:
```suggestion
          with open("/state/{{inputs.parameters.file-path}}") as f:
```